src/main/java/com/bisnode/opa/client/rest/UrlNormalizer.java

```diff
@@ -0,0 +1,23 @@
+package com.bisnode.opa.client.rest;
+
+import java.util.Optional;
+
+class UrlNormalizer {
+    public static Optional<String> normalize(String input) {
+        return Optional.ofNullable(input)
```

Comment:
I know it will break the code flow, but what do you think about comparing the input with the output of this method and, in case of mismatch, log a warning, that the supplied url is malformed?

Reply:
Nice idea